pkgs/top-level/all-packages.nix

```diff
@@ -16545,7 +16545,9 @@ in
     python = python3;
   };
 
-  libsemanage = callPackage ../os-specific/linux/libsemanage { };
+  libsemanage = callPackage ../os-specific/linux/libsemanage {
+    python = python3;
+  };
```

Comment:
can we do this in the expression?

Reply:
for most usages of python outside of python-modules, you generally want to parameterize python like this incase you need to override package versions, or if you want to test with a different version of python